actor/src/lib.rs

```diff
@@ -347,8 +347,8 @@ pub struct InitializedActor<A: Actor> {
 
 impl<A: Actor> InitializedActor<A> {
     /// Constructor.
-    pub fn new(actor: A, mailbox_capacity: usize) -> Self {
-        let (sender, receiver) = mpsc::channel(mailbox_capacity);
+    pub fn new(actor: A, mailbox_capacity: u32) -> Self {
+        let (sender, receiver) = mpsc::channel(mailbox_capacity as usize);
```

Comment:
I can't see why do we need this change, cause `InitializedActor` doesn't implement `Deserialize` and/or `Serialize`.
If something, that uses `InitializedActor`, also implements `Deserialize` and/or `Serialize`, then conversion should be done inside it